src/common/metrics/vaccinations.tsx

```diff
@@ -65,13 +65,24 @@ function renderStatus(projections: Projections): React.ReactElement {
   const peopleVaccinated = formatInteger(info.peopleVaccinated);
   const percentVaccinated = formatPercent(info.ratioVaccinated, 1);
 
+  let distributedText = <Fragment />;
+  if (info.dosesDistributed && info.ratioDosesAdministered) {
+    distributedText = (
+      <Fragment>
+        This this means that {locationName} has administered{' '}
```

Comment:
Two this's here.  Also, "this means that" feels a bit unnecessary - being pedantic, it doesn't really imply the % usage of total supply, it only really implies the total doses administered.  Could this just be `{locationName} has administered`?

Reply:
Gah, good catch on "This this". 😞 

"This means that" was bugging me too, so I removed it. But it is the copy that Debbie had approved.  But I think it's probably an improvement.